development/compiling/compiling_for_android.rst

```diff
@@ -8,147 +8,154 @@ Compiling for Android
-
-::
+The resulting APK will be located at ``bin/android_release.apk``.
 
-    bin\android_release.apk
-
--  Debug template (used when exporting with "Debugging Enabled" ON)
+-  Debug template (used when exporting with **Debugging Enabled** checked)
 
 ::
 
-    C:\godot> scons platform=android target=release_debug
-    C:\godot> cd platform/android/java
-    C:\godot\platform\android\java> gradlew build
-
-The resulting APK is in:
+    scons platform=android target=release_debug
+    cd platform/android/java
+    # On Windows
+    .\gradlew build
+    # On Linux/macOS
+    ./gradlew build
 
-::
 
-    bin\android_debug.apk
+The resulting APK will be located at ``bin/android_debug.apk``.
 
 Faster compilation
 ~~~~~~~~~~~~~~~~~~
 
-If you are on Unix or installed PyWin32 on Windows and have multiple CPU
-cores available, you can speed up the compilation by adding the ``-jX``
-argument to the SCons command, where ``X`` is the number of cores that you
-want to allocate to the compilation, e.g. ``scons -j4``.
-
+If you have multiple CPU cores available, you can speed up compilation
+by adding the ``-jX`` argument to the SCons command, where ``X`` is the
+number of cores you want to allocate to building, e.g. ``scons -j4``.
 
 Adding support for x86 devices
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you also want to include support for x86 devices, run the scons command
+If you also want to include support for x86 devices, run the SCons command
 a second time with the ``android_arch=x86`` argument before building the APK
-with Gradle. For example for the release template:
+with Gradle. For example, for the release template:
 
 ::
 
-    C:\godot> scons platform=android target=release
-    C:\godot> scons platform=android target=release android_arch=x86
-    C:\godot> cd platform/android/java
-    C:\godot\platform\android\java> gradlew build
+    scons platform=android target=release
+    scons platform=android target=release android_arch=x86
+    cd platform/android/java
+    # On Windows
+    .\gradlew build
+    # On Linux/macOS
+    ./gradlew build
 
 
-This will create a fat binary that works in both platforms, but will add
+This will create a fat binary that works on both platforms, but will add
 about 6 megabytes to the APK.
 
 Troubleshooting
 ~~~~~~~~~~~~~~~
 
 It might be necessary to clean the build cache between two APK compilations,
 as some users have reported issues when building the two export templates
-one after the other.
+one after the other. This can be done using the following command:
+
+::
+
+    scons platform=android -c
 
 Using the export templates
 --------------------------
 
-As export templates for Android, Godot needs release and debug APKs that
-were compiled against the same version/commit as the editor. If you are
-using official binaries for the editor, make sure to install the matching
-export templates, or to build your own from the same version.
+Godot needs release and debug APKs that were compiled against the same
```

Comment:
It would be good to refer to the "Exporting for Android" docs here. I haven't checked if they are explicit about how to select the architectures that should be included in the exported APK, but if not it should likely be added there and referenced here.

Reply:
The Exporting for Android page doesn't really explain it, so I'll skip that for now.